### ISSUE TITLE
The visitor's message text doesn't fit into the message box on iOS 10

### ISSUE THREAD

**Issue by aoriens** [zendesk/zendesk_sdk_chat_ios#108]
Hello!
I have a problem with the visitor's message text appearance under iOS 10 beta 4, iPhone 6, built by Xcode 8 beta 4. There is the same issue about agent's messages.
![img_0143](https://cloud.githubusercontent.com/assets/2458281/17435382/74d46ad8-5b21-11e6-8c18-4d071d335da9.PNG)

I decreased the `bubleInsets` and the `textInsets` properties, but the text still overflowed. Here is a fragment of my styling code:

~~~
  insets = UIEdgeInsetsMake(8.0f, 75.0f , 7.0f, 15.0f);
  [[ZDCVisitorChatCell appearance] setBubbleInsets:[NSValue valueWithUIEdgeInsets:insets]];
  insets = UIEdgeInsetsMake(12.0f, 15.0f, 12.0f, 15.0f);
  [[ZDCVisitorChatCell appearance] setTextInsets:[NSValue valueWithUIEdgeInsets:insets]];
  [[ZDCVisitorChatCell appearance] setBubbleBorderColor:[[UIColor whiteColor] colorWithAlphaComponent:0.3f]];
  [[ZDCVisitorChatCell appearance] setBubbleColor:[[UIColor blackColor] colorWithAlphaComponent:0.6f]];
  [[ZDCVisitorChatCell appearance] setBubbleCornerRadius:@(3.0f)];
  [[ZDCVisitorChatCell appearance] setTextAlignment:@(NSTextAlignmentLeft)];
  [[ZDCVisitorChatCell appearance] setTextColor:[UIColor whiteColor]];
  [[ZDCVisitorChatCell appearance] setTextFont:[UIFont systemFontOfSize:15.0f]];
  [[ZDCVisitorChatCell appearance] setUnsentTextColor:[UIColor colorWithWhite:0.26f alpha:1.0f]];
  [[ZDCVisitorChatCell appearance] setUnsentTextFont:[UIFont systemFontOfSize:12.0f]];
  [[ZDCVisitorChatCell appearance] setUnsentMessageTopMargin:@(5.0f)];
  [[ZDCVisitorChatCell appearance] setUnsentIconLeftMargin:@(10.0f)];
  
  insets = UIEdgeInsetsMake(8.0f, 55.0f, 7.0f, 15.0f);
  [[ZDCAgentChatCell appearance] setBubbleInsets:[NSValue valueWithUIEdgeInsets:insets]];
  insets = UIEdgeInsetsMake(12.0f, 15.0f, 12.0f, 15.0f);
  [[ZDCAgentChatCell appearance] setTextInsets:[NSValue valueWithUIEdgeInsets:insets]];
  [[ZDCAgentChatCell appearance] setBubbleBorderColor:[[UIColor blackColor] colorWithAlphaComponent:0.2f]];
  [[ZDCAgentChatCell appearance] setBubbleColor:[UIColor whiteColor]];
  [[ZDCAgentChatCell appearance] setBubbleCornerRadius:@(3.0f)];
  [[ZDCAgentChatCell appearance] setTextAlignment:@(NSTextAlignmentLeft)];
  [[ZDCAgentChatCell appearance] setTextColor:[[UIColor blackColor] colorWithAlphaComponent:0.8f]];
  [[ZDCAgentChatCell appearance] setTextFont:[UIFont systemFontOfSize:15.0f]];
  [[ZDCAgentChatCell appearance] setAvatarHeight:@(30.0f)];
  [[ZDCAgentChatCell appearance] setAvatarLeftInset:@(14.0f)];
  [[ZDCAgentChatCell appearance] setAuthorColor:design.color3];
  [[ZDCAgentChatCell appearance] setAuthorFont:[UIFont systemFontOfSize:12]];
  [[ZDCAgentChatCell appearance] setAuthorHeight:@(25.0f)];
~~~

**Comment by mathewcropper** [comment]
Hi @aoriens. Thanks for passing this on. We're aware of this and a handful of other issues with the Chat SDK when running on iOS 10.

We are working on an update in our next sprint. This will be focused on dealing with iOS 10 compatibility.

On Monday, I'll be publishing a post in our [SDK community](https://support.zendesk.com/hc/en-us/community/topics/200488257-Zendesk-SDKs) and updating our documentation here to make it clearer what the various known issues are.

**Comment by mathewcropper** [comment]
Hi @aoriens. We just released v1.3.1.1 of the Chat SDK, with fixes for issues relating to iOS 10 (beta 4).

I'm going to close this issue, but if you come across anything else please do raise it. Thanks for your patience!

**Comment by aoriens** [comment]
The problem has disappeared in v1.3.1.1. Thanks!
